### DISCUSSION TITLE
Buying blank (bootloaderless) LGT8F328P

### DISCUSSION THREAD

**Discussion by sengiv** [dbuezas/lgt8fx#158] [General]
Hi,
Has anyone managed to purchase blank chips directly from LGT or distributor?

So far I've only been able to get my hands on minimum system boards with bootloaders. (Ebay/Aliexpress)
Many big suppliers (Element14/digikey), don't carry LGT stuff.

Why? (if anybody is wondering)
A few advantages
- Lower cost
- Integrating into existing designs

**Comment by LaZsolt** [comment]
You can buy here: https://www.aliexpress.com/item/32821636647.html

**Comment by sengiv** [comment]
Thanks!!
I must have overlooked this listing

**Comment by LaZsolt** [comment]
https://www.alibaba.com/product-detail/LGT8F328P-LQFP32-LQFP48-replace-ATMEGA328P-chip_1600117108332.html

**Comment by pixelwaster** [comment]
I was just going to post that I got my 328P-48 chips. 

When ordering from the Aliexpress store, don't get a mixed order of 32 and 48 pin chips. Someone on on of the Russian forums had some trouble getting only one type when ordering both. It looks like a Russians and I bought the first 2 of 3 sales of chips. The price has double since my order, Only found out about the price when I went to file a never got it, 3 months on an airplane!! from the tracking info. Going to try a bootloader and blink hope fully tomorrow,  Don't forget to get QFP to DIP carrier boards to use on a breadboard,

Is there an "official" Arduino-ish pin naming somewhere? I tried the pins files but couldn't see to make them come out correct
